### ISSUE TITLE
MLM Pre-training Code Version

### ISSUE THREAD

**Issue by robinsongh381** [microsoft/DeBERTa#75]
Hello !
Thank you for sharing a great piece of work.

I was wondering whether the MLM [pre-training code ](https://github.com/microsoft/DeBERTa/blob/master/experiments/language_model/mlm.sh)is for training DeBERTa v3 or v2 ? (or v1)

Regards

**Comment by R4ZZ3** [comment]
I am also waiting for v3 pretraining codes and would absolutely LOVE to see it integrated to huggingface FLAX !

**Comment by BigBird01** [comment]
It's for v1 &v2, we are still working on the v3 code, will release it once we passed the internal processing.

**Comment by Oxi84** [comment]
> It's for v1 &v2, we are still working on the v3 code, will release it once we passed the internal processing.

I do not get it?
Why do you work on these models when they just give random predictions.
You do not need to work on models at all, it would me much faster to just to replace [MASK] with random(vocabulary) and you would get the same results with the same extremely bad accuracy.

@BigBird01
